text/0034-telemetry-policies.md

```diff
@@ -114,7 +115,12 @@ The collected data is sent to [Segment](https://segment.com/). Segment is a plat
 | `sortable_attributes.has_geo`           | `true` if `_geo` is set as a sortable attribute, otherwise `false` | true | `Settings Updated`, `SortableAttributes Updated` |
 | `filterable_attributes.total`           | Number of filterable attributes | 3 | `Settings Updated`, `FilterableAttributes Updated` |
 | `filterable_attributes.has_geo`         | `true` if `_geo` is set as a filterable attribute, otherwise `false` | false | `Settings Updated`, `FilterableAttributes Updated`|
-| `searchable_attributes.total`           | Number of searchable attributes | 4 | `Settings Updated`, `SearchableAttributes Update` |
+| `searchable_attributes.total`           | Number of searchable attributes | 4 | `Settings Updated`, `SearchableAttributes Updated` |
+| `typo_tolerance.enabled`                 | Whether the typo tolerance is enabled | `true` | `Settings Updated`, `TypoTolerance Updated` |
+| `typo_tolerance.disable_on_attributes_used    | `true` if at least one value is defined | `false` | `Settings Updated`, `TypoTolerance Updated` |
+| `typo_tolerance.disable_on_words_used   | `true` if at least one value is defined | `false` | `Settings Updated`,  `TypoTolerance Updated` |
```

Comment:
The specification of the typo tolerance allows to find this information, no? I would like to avoid duplicating information. Maybe some links would do the trick here!